extras/mmu_sensors.py

```diff
@@ -93,7 +93,7 @@ def note_filament_present(self, is_filament_present):
 
     def _process_state_change(self, eventtime, is_filament_present):
         # Determine "printing" status
-        is_printing = self.printer.lookup_object("idle_timeout").get_status(eventtime)["state"] == "Printing"
+        is_printing = self.printer.lookup_object("print_stats").get_status(eventtime)["state"] == "printing"
```

Comment:
This change is un-related to the respooler rename. It is in a separate commit and can put it in a separate PR if preferred.

Reply:
want be to pull and add idle_timeout fallback?